examples/ecp5_evn/Makefile

```diff
@@ -6,15 +6,15 @@ all: ${PROJ}.bit
 	yosys -p "synth_ecp5 -noccu2 -nobram -nomux -json $@" $<
 
 %_out.config: %.json
-	nextpnr-ecp5 --json $< --basecfg ../../misc/basecfgs/empty_lfe5um5g-85f.config --textcfg $@ --um5g-85k --package CABGA381 --lpf ecp5evn.lpf
+	nextpnr-ecp5 --json $< --basecfg /usr/share/prjtrellis/misc/basecfgs/empty_lfe5um5g-85f.config --textcfg $@ --um5g-85k --package CABGA381 --lpf ecp5evn.lpf
```

Comment:
Please keep the relative paths as they work for everyone regardless of how/where the tools are installed, long term plan is to remove the baseconfigs altogether.